packages/components/src/components/error-modal/__tests__/error-modal-content.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ErrorModalContent from '../error-modal-content';
+
+describe('<ErrorModalContent />', () => {
+    let modal_root_el: HTMLDivElement;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+    const error_message = [{ message: 'Sorry for the interruption' }];
+    it('should renders the error message', () => {
+        render(<ErrorModalContent error_message={error_message} />);
+        const error_description = screen.getByText('[object Object]');
```

Comment:
@hamza-deriv what is this?

Reply:
When i am trying to mock the Error content so it is showing as [object Object] instead of the message, so i get by Text is there any other 
way to check it?

Reply:
Refactored